packages/core/src/test/amazonq/customizationUtil.test.ts

```diff
@@ -82,16 +82,39 @@ describe('CodeWhisperer-customizationUtils', function () {
         assert.strictEqual(actualCustomization.name, selectedCustomization.name)
     })
 
-    it('Returns AB customization', async function () {
-        sinon.stub(AuthUtil.instance, 'isValidEnterpriseSsoInUse').returns(true)
+    it(`setSelectedCsutomization should set correctly if override is false or not specified`, async function () {
```

Comment:
nit: 
```suggestion
    it(`setSelectedCustomization should set correctly if override is false or not specified`, async function () {
```

Reply:
What is this test case trying to illustrate? It's not really clear from the title

Reply:
so when override is true, it's saying the setCustomization is invoked by us and not by users. In such case, the customization we pass in the call `setCustomization` is from A/B service, and we will override users' own selection once for each customization arn provided by the service.

Reply:
when override is false, it means the setCustomization is invoked by users, so we will respect to users' selection anyway